pkgs/development/python-modules/rstcheck/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, fetchPypi, buildPythonPackage, docutils }:
+
+buildPythonPackage rec {
+  pname = "rstcheck";
+  version = "3.3.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "92c4f79256a54270e0402ba16a2f92d0b3c15c8f4410cb9c57127067c215741f";
+  };
+
+  doCheck = false;
```

Comment:
Could you leave a comment with the reason for disabling them?

Reply:
The test isn't bundled in the PyPi tarball, we would have to pull the module from the GitHub repository to execute the tests afaik.

Reply:
We do actually frequently do just that to enable the tests, but if we're not going to could you leave a comment _in the code_ here, e.g. simply `# tests not included in pypi tarball`

Reply:
I've switched to github, got tests executing 
![image](https://user-images.githubusercontent.com/86573128/131731405-14842857-0a7f-4cea-b681-5bd4b5d9c10e.png)

when pulling from Git, I specify a tagged version instead of a revision hash, is this ok?

```nix
  version = "v3.3.1";

  src = fetchFromGitHub {
    rev = version;
  };```